content/Watsonx/WatsonxGov/Level-4/203.md

```diff
@@ -0,0 +1,155 @@
+---
+title: '203: Configure the governance console & monitoring'
+timeToComplete: 45
+updated: 2024-06-20
+---
+
+# 203: Configure the governance console & monitoring
+
+In this lab We will enable the watsonx.governance console (OpenPages) integration with Factsheets and the watsonx.governance monitoring service (OpenScale), and load sample data into the governance console.
+
+## Prerequisites
+
+This lab depends on the setup in [Lab 201](/watsonx/watsonxgov/level-4/201), which means having a running CP4D environment from TechZone.
+
+## Create a user
+
+In this step, you will create a sample user in Cloud Pak for Data. This step is required for the rest of the hands-on labs.
+
+1. First log in to Cloud Pak for Data using the *Console Route*, *Username*, and *Password* you got from the *update-configmap-success* CP4D pipeline logs in [Lab 201](/watsonx/watsonxgov/level-4/201).
+2. From the home screen, click the **hamburger menu** in the top left, then under **Administration**, click **Access control**. <br />![access control](../images/203/watsonxgov-l4_access-control.webp)
+3. Click the blue **Add users** button, and then fill out the profile information. Use `complianceofficer` for the username, as this will be referenced in future labs. Make sure not to forget your password! Click **Next**. <br />![user details](../images/203/watsonxgov-l4_user-details.webp)
```

Comment:
This screenshot should be updated to show the `complianceofficer` username to be consistent